### ISSUE TITLE
CBOR support?

### ISSUE THREAD

**Issue by nphippsoneill** [eclipse/leshan#1433]
### Question

I couldn't find this answer in supported features or any other question. Does the leshan Demo support CBOR?

**Comment by jvermillard** [comment]
This could help https://github.com/eclipse/leshan/wiki/LWM2M-1.1-supported-features

**Comment by npon97** [comment]
Yes that's helpful, just want to clarify if the Demo supports all the features that the API supports?

**Comment by sbernard31** [comment]
>  Does the leshan Demo support CBOR on CoAPS?

It should.

>  just want to clarify if the Demo supports all the features that the API supports?

Demo does not expose all feature available in the API. (it's just a subset)
